discovery/gossiper_test.go

```diff
@@ -1498,7 +1501,8 @@ func TestSignatureAnnouncementRetryAtStartup(t *testing.T) {
 		NotifyWhenOffline:    ctx.gossiper.reliableSender.cfg.NotifyWhenOffline,
 		Router:               ctx.gossiper.cfg.Router,
 		TrickleDelay:         trickleDelay,
-		RetransmitDelay:      retransmitDelay,
+		RetransmitTicker:     ticker.NewForce(retransmitDelay),
```

Comment:
Nit: can be squashed with the previous commit to allow the tests to build.

Reply:
Done!